config/locales/views/investments/zh-CN.yml

```
@@ -0,0 +1,17 @@
+---
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)